keyboards/xbows/nature/Peripherals.c

```diff
@@ -0,0 +1,177 @@
+/* Copyright 2020 Shulin Huang <mumu@x-bows.com>
```

Comment:
This seems like a bodge to work round not going through with the requested changes within #10707?

Reply:
yes

Reply:
In which case, this should be removed then. #10707 just needed some small changes, im not sure why you closed it. This just adds fragmentation.

Reply:
I modified file rgb_matrix_drivers.c  ,#10860

Reply:
If the file goes through, I will re-upload my keyboard code.